lib/browser/mqtt.ts

```diff
@@ -56,6 +56,34 @@ export type MqttConnectionError = (error: CrtError) => void;
  */
 export type MqttConnectionInterrupted = (error: CrtError) => void;
 
+/**
+ * Listener signature for event emitted from an {@link MqttClientConnection} when the connection has been
+ * connected successfully.
+ *
+ * @param session_present A boolean indicating if the connection resumed a session
+ *
+ * @category MQTT
+ */
+export type MqttConnectionSuccess = (session_present: boolean) => void;
```

Comment:
Note: MQTT.js doesn't provide a return code from the server upon a successfully connect, so we do not expose it here. NodeJS/Desktop does expose it, and so we return it there, but the browser doesn't really have an equivalent.